src/test/java/io/stargate/sgv2/jsonapi/api/v1/util/DataApiCommandSenders.java

```diff
@@ -0,0 +1,8 @@
+package io.stargate.sgv2.jsonapi.api.v1.util;
+
+/** Helper class used for constructing and sending commands to the Data API */
+public class DataApiCommandSenders {
+  public static DataApiTableCommandSender tableCommand(String namespace, String tableName) {
```

Comment:
Makes little sense until adding `namespaceCommand()` (or should that be `keyspaceCommand()`), but that will be added soon.